.github/workflows/test_rust.yml

```diff
@@ -64,6 +64,12 @@ jobs:
           command: fmt
           args: --all -- --check
 
+      - name: Lint AS3 playerglobals 
+        uses: actions-rs/cargo@v1
```

Comment:
Let's not run this on entire matrix, linux-only or even linux-stable-only should be enough.